script.js

```diff
@@ -0,0 +1,70 @@
+const addbookBtn = document.querySelector('#add-book');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookList = JSON.parse(localStorage.getItem('bookShop')) || [];
+
+const addBook = (bookTitle, bookAuthor) => {
+  const book = {};
+  book.title = bookTitle.value;
+  book.author = bookAuthor.value;
+  bookList.push(book);
+
+  bookTitle.value = '';
+  bookAuthor.value = '';
+};
+
+const displayBook = (book) => {
+  const { title, author } = book;
+  const bookContainer = document.createElement('div');
+  bookContainer.className = 'book';
+  bookContainer.innerHTML = `<span class='title'>${title}</span><br>
+    <span class='by'>by</span><br>
+    <span class='author'>${author} </span><br>
+    <button class='delete'>Remove</button>
+    <hr>`;
+  return bookContainer;
+};
+
+const displayBooks = () => {
+  const currentBooks = document.querySelector('#books-list');
+  const libraryContainer = document.createElement('div');
+
+  bookList.forEach((book) => {
+    libraryContainer.appendChild(displayBook(book));
+    currentBooks.innerHTML = libraryContainer.innerHTML;
+  });
+};
+
+addbookBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (bookTitle.value === null || bookAuthor.value === null) {
+    return;
+  }
+  addBook(bookTitle, bookAuthor);
+  localStorage.setItem('bookShop', JSON.stringify(bookList));
+
+  displayBooks();
+});
+
+window.onload = displayBooks();
+
+// Remove Book
+const removeBook = (button) => {
+  const bookList = JSON.parse(localStorage.getItem('bookShop'));
+  const parentDiv = button.parentNode;
+  const myTitle = parentDiv.querySelector('.title').textContent;
+  const myAuthor = parentDiv.querySelector('.author').textContent;
+  const booksLeft = bookList.filter(
+    (book) => book.title !== myTitle && book.author !== myAuthor,
+  );
+  window.localStorage.setItem('bookShop', JSON.stringify(booksLeft));
+
+  parentDiv.remove();
+};
```

Comment:
- Nice work on the project so far. However, there's one issue I've found:
   - [ ] Currently when you try to remove a book and then add another one, all of the books are displayed, even the removed one. Kindly fix this behavior.
   
   [Hint] This behavior is likely caused by your `removeBook` function, where you actually remove the book from the DOM and the local storage. However, you don't change the `bookList` array where you display the books from.